examples/tcpsynbl.yaml

```diff
@@ -0,0 +1,30 @@
+programs:
+  - name: tcpsynbl
+    metrics:
+      histograms:
+        - name: tcp_syn_backlog
+          help: trace the TCP SYN backlog size
+          table: bucket
+          bucket_type: linear
+          bucket_min: 0
+          bucket_max: 1024 # `sysctl net.ipv4.tcp_max_syn_backlog`
+          bucket_multiplier: 1
```

Comment:
This produces 1024 time series, which is a bit excessive.

You can get a decent approximation of the backlog with:

```yaml
bucket_min: 0
bucket_max: 20
bucket_multiplier: 50
```

Reply:
You're right. Thank you for pointing this out

✅ https://github.com/cloudflare/ebpf_exporter/pull/76/commits/dee0154db5fbc98542a0779c8352075601b89f31